modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/baseComponents/BaseAPIApplicationFields.tsx

```diff
@@ -129,6 +142,10 @@ export default function BaseAPIApplicationFields({
 					id="modalURLField"
 					onChange={({target: {value}}) => {
 						setUserEditedURL(true);
+						setBaseURLContent({
+							errorMessage: baseURLEnterValidErrorMessage,
```

Comment:
I think that you could inline this

Reply:
Yes, you are right. This is reminiscent of a different implementation. I will fix it. Thanks!